sql/rename.go

```diff
@@ -203,6 +203,88 @@ func (p *planner) RenameIndex(n *parser.RenameIndex) (planNode, error) {
 
 	idx.Name = newIdxName
 	descKey := MakeDescMetadataKey(tableDesc.GetID())
+	if err := tableDesc.Validate(); err != nil {
```

Comment:
@tamird how do i test that? I runned a `select` after rename the column, which test that indirectly.
Suggestions?